src/components/Dropdown.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import Touchable from 'src/components/Touchable'
+import DownArrowIcon from 'src/icons/DownArrowIcon'
+import UpArrowIcon from 'src/icons/UpArrowIcon'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props<T> {
+  options: { value: T; label: string }[]
+  onValueSelected(value: T): void
+  testId?: string
+}
+
+/* Dropdown component with unscrollable list of options */
+
+function Dropdown<T>(props: Props<T>) {
+  const testID = props.testId ? props.testId : 'Dropdown'
+  const [isOpen, setIsOpen] = useState(false)
+  const [labelSelected, setLabelSelected] = useState<string | undefined>()
+
+  const toggleOpen = () => {
+    setIsOpen(!isOpen)
+  }
+
+  return (
+    <View style={{ zIndex: 100 }} testID={testID}>
+      <Touchable onPress={toggleOpen} testID={testID + '-Touchable'}>
+        <View style={styles.selectedOptionContainer}>
+          <Text style={styles.optionText}>{labelSelected}</Text>
+          {!isOpen ? (
+            <DownArrowIcon color={Colors.primary} strokeWidth={2} />
+          ) : (
+            <UpArrowIcon color={Colors.primary} strokeWidth={2} />
+          )}
+        </View>
+      </Touchable>
+      {isOpen && (
+        <View>
+          <View style={styles.optionsContainer}>
+            {props.options.map((option) => {
+              return (
+                <Touchable
+                  style={{ padding: Spacing.Small12 }}
+                  onPress={() => {
+                    setLabelSelected(option.label)
+                    toggleOpen()
+                    props.onValueSelected(option.value)
+                  }}
+                  testID={testID + '-' + option.label}
+                  key={option.label}
+                >
+                  <Text style={styles.optionText}>{option.label}</Text>
+                </Touchable>
+              )
+            })}
+          </View>
+        </View>
+      )}
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  selectedOptionContainer: {
+    padding: Spacing.Small12,
+    borderColor: Colors.gray2,
+    borderRadius: Spacing.Tiny4,
+    borderWidth: 1.5,
```

Comment:
maybe this can just be 1? 😅 i'm not sure that .5 px renders well anyway

Reply:
Just reusing the `messageTextInput` style that was used in TokenImport.
But I can use 1